src/components/Flyout/FlyoutWindow.part.tsx

```diff
@@ -352,11 +352,15 @@ export class FlyoutWindowInt extends React.Component<FlyoutWindowProps, FlyoutWi
   }
 
   private setFlyoutRef = (el: HTMLDivElement) => {
+    if (this.flyoutContainer) {
+      this.flyoutContainer.removeEventListener('scroll', this.onScroll);
+    }
+
     if (el) {
       this.flyoutContainer = el;
-      this.flyoutContainer.removeEventListener('scroll', this.onScroll);
-      this.flyoutContainer.addEventListener('scroll', this.onScroll);
     }
+
+    this.flyoutContainer.addEventListener('scroll', this.onScroll);
```

Comment:
I'm not sure why the proposed logic was not taken. This one is not symmetric and not as robust as the proposed one.

Can you explain why this version?

Reply:
Both setting container and adding listener should occur only if `el` was passed. I've updated the code

Reply:
But if the last time something was passed in `el` was `null` it should be reflected via the global container. Otherwise you

- still have the ref that needs to be considered on the GC (just unnecessary load on the GC as the ref count is still 1, where it could be 0 improving the gen. cleanup)
- reflects a false state of the DOM in the component (even though it should only be `null` when the component unmounts it could still be used)

The logic is just more sound this way. It "somewhat" works already as currently implemented.